### ISSUE TITLE
ColumnSet touch move issue

### ISSUE THREAD

**Issue by Egor-Chib** [SitePen/dgrid#1357]
On touch enabled devices, once get created, dgrid with mixed in ColumnSet performs not intended horizontal scrolling on mouse move if mouse pointer is within client rect of the grid.

In the module ColumnSet popery _currentlyTouchedColumnSet has no declaraion and no initialization. Therefore it is "undefined". 

Function horizTouchMove in module ColumnSet has following code:
```
				on(target, getTouchEventName('move'), function (event) {
					if (grid._currentlyTouchedColumnSet === null) {
						return;
					}
```
Notice that "_currentlyTouchedColumnSet === **null**"  - it is being compared with NULL. But initial property value is "undefined". Therefore right after grid creation it is in "horizontal move enabled" mode on touch enbled devices.

**Comment by Egor-Chib** [comment]
When I say "touch enabled device" we should concider the fact, that under Chrome all laptops are "touch enabled". Use chrome to reproduce it, probably use laptop or desktop with touch screen.

**Comment by Egor-Chib** [comment]
Obvious workaround it to set property to null
grid._currentlyTouchedColumnSet === null
It helps in the first moment.

Unfortunately, if user attepts to use touch move, using touch screen and finger touch, event "end" (which is expected in the code), not always happen, and grid will be left in "touch move ON" mode even after you finish you finger touch move. And therefore grid, again, will react on mouse move and do unwanted scroll.

**Comment by dylans** [comment]
@msssk any thoughts on whether or not this is the right approach?

**Comment by edhager** [comment]
@Egor-Chib, you are correct.  The null check was too restrictive.  I pushed a fix to 87e02af29efba5d7c83f50604b3c9a1a1c70140a.

Thanks!
